client/components/account-status/index.js

```diff
@@ -0,0 +1,120 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import {
+	Button,
+	Card,
+	CardBody,
+	CardHeader,
+	FlexBlock,
+	FlexItem,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AccountFees from './account-fees';
+import AccountStatusItem from './account-status-item';
+import DepositsStatus from 'components/deposits-status';
+import PaymentsStatus from 'components/payments-status';
+import StatusChip from './status-chip';
+import './style.scss';
+import './shared.scss';
+
+const AccountStatusCard = ( props ) => {
+	const { title, content, children, value } = props;
+	return (
+		<Card isMedium>
+			<CardHeader
+				className={ 'woocommerce-account-status__header' }
+				direction={ 'row' }
+				align={ 'center' }
+				justify={ 'left' }
+				gap={ 3 }
+				expanded
+			>
+				{ title }
+			</CardHeader>
+			<CardBody>{ children || value || content }</CardBody>
```

Comment:
Any reason why we are supporting `value` and `content` props as well? IMO, this might create inconsistent in the code as some devs might use different props to send card body data to this component, what do you think?

Reply:
Thanks for pointing this. I didn't update it after transition was done.

Reply:
Regarding `value` - it is one of the React props which can be provided to the component base on its usage.

Reply:
>it is one of the React props which can be provided to the component base on its usage

Just so that I understand your point of view better, what can be the use cases where we'll prefer `value` over `children` prop.